validatedeps.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
```

Comment:
yes I think so - optional args passed to it perhaps? 

Perhaps @sophistifunk can suggest how to do it, given he first created checkdeps.js IIRC